pulpcore/cli/core/context.py

```diff
@@ -379,24 +321,20 @@ class PulpTaskGroupContext(PulpEntityContext):
     ENTITY = _("task group")
     ENTITIES = _("task groups")
     HREF = "task_group_href"
-    LIST_ID = "task_groups_list"
-    READ_ID = "task_groups_read"
+    ID_PREFIX = "task_groups"
 
 
 class PulpUploadContext(PulpEntityContext):
     ENTITY = _("upload")
     ENTITIES = _("uploads")
     HREF = "upload_href"
-    LIST_ID = "uploads_list"
-    READ_ID = "uploads_read"
-    CREATE_ID = "uploads_create"
-    UPDATE_ID = "uploads_update"
-    DELETE_ID = "uploads_delete"
-    COMMIT_ID: ClassVar[str] = "uploads_commit"
+    ID_PREFIX = "uploads"
+    # TODO: Not so nice
+    PARTIAL_UPDATE_ID = "uploads_update"
```

Comment:
This works, but i am unsure whether it's a sign that we should not use `update()`, but implement a `upload_chunk()` on this context.

Reply:
`upload_chunk` it is.